src/WebJobs.Extensions.DurableTask/OrchestrationStatusQueryCondition.cs

```diff
@@ -40,5 +55,15 @@ public class OrchestrationStatusQueryCondition
         /// ContinuationToken of the pager.
         /// </summary>
         public string ContinuationToken { get; set; }
+
+        /// <summary>
+        /// Return orchestration instances that have this instance id prefix.
+        /// </summary>
+        public string InstanceIdPrefix { get; set; }
+
+        /// <summary>
+        /// Determines whether the query will include the input of the orchestration.
+        /// </summary>
+        public bool FetchState { get; set; } = false;
```

Comment:
Since this is the orchestration query condition, this should be named `FetchInput`.